project_event/models/res_partner_category.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# © 2018 Savoir-faire Linux
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class PartnerCategory(models.Model):
+    _name = 'res.partner.category'
+    _inherit = ['res.partner.category']
```

Comment:
You don't need to specify the name if you are already inheriting and extending, only if you want to create a new table. It's ok, but you could remove this line